### PR TITLE
multimodule working

### PR DESCRIPTION
**Pull request by sorenmacbeth**

### CODE REVIEW

**Comment by sorenmacbeth** [comment]
The version number of elephantdb-cascading is ahead of the other modules, not sure what we should do about that, or if it even matters.

**Comment by sritchie** [comment]
Let's bump all to 0.4.0 and make the api change.

File: elephantdb-cascading/.gitignore [hunk 1]
```diff
@@ -0,0 +1,14 @@
```
Comment (sritchie) [inline elephantdb-cascading/.gitignore]: Can we have this at the root only?

File: elephantdb-cascading/.travis.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# See this guide for more settings:
```
Comment (sritchie) [inline elephantdb-cascading/.travis.yml]: Travis for the root only, and have it run "lein sub test" to run ALL tests for everyone

File: elephantdb-cascading/LICENSE [hunk 3]
```diff
@@ -0,0 +1,19 @@
```
Comment (sritchie) [inline elephantdb-cascading/LICENSE]: License at the root only.

File: elephantdb-cascading/README.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+Tap and pipe assembly for writing to elephantdb via Cascading.
```
Comment (sritchie) [inline elephantdb-cascading/README.md]: Remove this for now, we'll fix the versions in the main README.

**Pull request merged**